### PR TITLE
MP3 and FLAC Support

### PR DESCRIPTION
**Pull request by avivbeeri**

### CODE REVIEW

File: src/modules/audio.c [hunk 1]
```diff
@@ -70,6 +99,8 @@ AUDIO_allocate(WrenVM* vm) {
     SDL_FreeWAV((uint8_t*)tempBuffer);
   } else if (data->audioType == AUDIO_TYPE_OGG) {
     free(tempBuffer);
+  } else if (data->audioType == AUDIO_TYPE_MP3) {
+    free(tempBuffer);
   }
```
Comment (Curanos) [inline src/modules/audio.c:104]: Shouldn't there a case for FLAC, too?

Comment (avivbeeri) [reply src/modules/audio.c:104]: You're right. I was just adding it!

Comment (Curanos) [reply src/modules/audio.c:106]: Ah yes, just a couple of minutes ago :-)
Sorry about that.

Comment (avivbeeri) [reply src/modules/audio.c:106]: Not at all, I appreciate the oversight. I wrote/merged a lot of code today so mistakes are likely.

**Pull request merged**